### PR TITLE
Fix compile warnings in y4m header parsing

### PR DESCRIPTION
**Pull request by Spacefish**
Replace warnings by error checking via assertions

### CODE REVIEW

File: Source/App/EbAppInputy4m.c [hunk 1]
```diff
@@ -36,13 +38,13 @@ int32_t readY4mHeader(EbConfig_t *cfg){
             continue;
         switch (*tokstart++) {
         case 'W': /* width, required. */
-            width = strtol(tokstart, &tokend, 10);
+            width = strtol((const char*)tokstart, (char**)&tokend, 10);
```
Comment (kylophone) [inline Source/App/EbAppInputy4m.c]: To avoid all this casting, `tokstart` and `tokend` should just be initialized as `const char*`.

**Comment by kylophone** [comment]
Thanks. Made one comment inline.

**Comment by Spacefish** [comment]
Good point i changed the datatype ;)

**Comment by kylophone** [comment]
Looks good. Thanks!

**Pull request merged**